### DISCUSSION TITLE
What does quarantine make?

### DISCUSSION THREAD

**Discussion by DimitriDR** [Homebrew/discussions#3351] [Everyday usage] [answered]
Title.
It is said in the man page:

> --[no-]quarantine: Disable/enable quarantining of downloads (default: enabled).

Nothing more is specified.

**Comment by SMillerDev** [comment, marked as answer]
It’s the quarantine feature on macOS: https://support.apple.com/en-us/HT202491

**Comment by DimitriDR** [comment]
It's as simple as that. It means that when `--no-quarantine` is specified, installed app overrules this mechanism? Because some apps (for instance, [Linear Mouse](https://linearmouse.org/) asks to put this argument).

**Reply by SMillerDev** [reply]
Because some developers don't know how to sign their apps correctly. So when homebrew sets the quarantaine flags, like is expected for downloading software on macOS, the OS complains about the software being unsigned or improperly signed. And that makes such developers unhappy.

**Reply by DimitriDR** [reply]
Okay, I see. So I should set this flag only on trusted apps. Makes sense, thanks!
